controller/handler_ctrl/session_request.go

```diff
@@ -65,25 +65,17 @@ func (h *sessionRequestHandler) HandleReceive(msg *channel2.Message, ch channel2
 					responseMsg.Headers[int32(k)] = v
 				}
 
-				if startXgressSession, err := h.r.Control.SendAndWaitWithTimeout(responseMsg, time.Second*10); err != nil {
-					log.Errorf("unable to respond (%s)", err)
-					h.network.RemoveSession(session.Id, true)
-				} else {
-					if startXgressSession.ContentType == int32(ctrl_pb.ContentType_StartXgressType) {
-						if err = h.network.StartSessionEgress(session.Id); err != nil {
-							log.WithError(err).Error("failure starting xgress")
-							h.network.RemoveSession(session.Id, true)
-						}
-					} else {
-						log.Errorf("unexpected reply to dial response: %v", startXgressSession.ContentType)
-						h.network.RemoveSession(session.Id, true)
```

Comment:
In addition to simplifying the logic, I bet this will make our session startup a little snappier, too.